editions/tw5.com/tiddlers/system/tw5.com-styles.tid

```diff
@@ -197,3 +198,24 @@ div.content {
 .tc-btn-download:active {
 	background: #1475ff;
 }
+
+/* WikiText rules */
```

Comment:
One big problem is that the formatting breaks down entirely when viewed at mobile sizes. I really think that it is essential that the cheat sheet is usable on a small screen.

Reply:
**I'll create a completely new PR for this issue.** 

That's a general problem with TW tables. This problem has always been there, the PR only makes it visible. 

There are 2 main reasons.

 1. For small screens we need:  `word-wrap` to be set to `word-wrap: anywhere` within tables. 
 2. default margins for `blockquotes, OL-LI and UL-li` need to be smaller in tables too. Especially if they are nested as shown in the examples here. They create too much white space.

Reply:
see:  **fix table overflow for small screens** #7010